### PR TITLE
Add 2023 User Survey blog post

### PR DESCRIPTION
**Pull request by annajung**
- Add 2023 User Survey blog post

cc @jbottum @johnugeorge @james-jwu @zijianjoy

### CODE REVIEW

**Comment by annajung** [comment]
deploy preview: https://deploy-preview-135--infallible-murdock-d05108.netlify.app/kubeflow-user-survey-2023/

File: _posts/2023-07-26-kubeflow-user-survey-2023.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+users to unlock the potential of machine learning. If you’re interested in becoming a Kubeflow contributor, please feel
+free to check out the resources below. We look forward to working with you!
+
+- Visit our [Kubeflow website](https://www.kubeflow.org/) or [Kubeflow GitHub Page](https://github.com/kubeflow)
+- Join
+  the [Kubeflow Slack](https://join.slack.com/t/kubeflow/shared_invite/enQtMjgyMzMxNDgyMTQ5LWUwMTIxNmZlZTk2NGU0MmFiNDE4YWJiMzFiOGNkZGZjZmRlNTExNmUwMmQ2NzMwYzk5YzQxOWQyODBlZGY2OTg)
+  workspace
+- Join the [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss) mailing list
+- Attend a [weekly community meeting](https://www.kubeflow.org/docs/about/community/)
+
```
Comment (zijianjoy) [inline _posts/2023-07-26-kubeflow-user-survey-2023.md:175]: I think referring to https://www.kubeflow.org/docs/about/community/ can provide people with more up-to-date information. We don't need to duplicate information here, which might change in the future.

Comment (annajung) [reply _posts/2023-07-26-kubeflow-user-survey-2023.md]: done, removed all links and pointed to the community page

**Comment by zijianjoy** [comment]
/lgtm
/approve

Awesome work, Anna!

**Comment by jbottum** [comment]
line 17, The survey consisted 21 questions... should this be , The survey consisted of 21 questions ?

**Comment by jbottum** [comment]
line 37, "52% of the users use the raw manifest installation to install Kubeflow " might be "52% use the raw manifest installation to install Kubeflow"

**Comment by jbottum** [comment]
graphic on geographic locations of users.  nit - labels for US and for Europe have some text that is cut off.

**Comment by jbottum** [comment]
Line 55, "we started to see an increase in the number of users adopting Kubeflow in the production environment". might change to "their production environment".

**Comment by jbottum** [comment]
line 91, If you are interested in improving Kubeflow documentation and forming a Kubeflow Documentation Team, join us at the next community meeting to introduce yourself to the community.    Comment - We have not discussed forming a Kubeflow Documentation Team.   I think its a good idea, but probably should float it in a community meeting.   Additionally, I might have been a little more aggressive and add or reword i.e.   We are asking that all companies, who are using Kubeflow,  to please proactively contribute at least one Pull Request on documentation in Kubeflow 1.8.   We request that you please add your PRs to the appropriate Working Group Meeting notes, and review it in the next Working Group meeting.

**Comment by jbottum** [comment]
Line 111, Kubeflow probably should be capitalized

**Comment by jbottum** [comment]
Line 115, I am proposing that we cut the sentence to end with :"upgrades take a great amount of effort for the users"/

This following detail (from 116 one) is valuable and we need to work on it.   I am glad to discuss it in community and working group meetings.  that said I propose that this might be cut from the blog. 
  
upgrades take a great amount of effort for the users without guaranteeing that upgrading Kubeflow will not introduce breaking changes to their current setup. One survey respondent
expressed, *“It's currently not possible to guarantee that upgrading kubeflow versions will work. We need to delete the
entire cluster, database, and all the resources just to re-install from scratch. This generally results in losing data (
because nothing guarantees that the DB scheme will be compatible from one version to another). There should be an easy
way to upgrade kubeflow.”*

**Comment by jbottum** [comment]
line 131. With a lack of support for both of these tools, users are expressing concerns about the challenges they face
in managing and tracking their models.   Perhaps this could be soften a little ..

The community recognizes the value of both of these tools for managing and tracking models and plans to introduce issues and PR to add these integrations.

**Comment by jbottum** [comment]
line 139, perhaps change "unofficial" security team to Kubeflow Security Team.

**Comment by jbottum** [comment]
line 146, Currently, all working groups are working towards the 1.8 release, planned to release
on [October 4th,.    maybe.   The Kubeflow Working Groups are actively developing the Kubeflow 1.8 release, whose releaes date is ...

**Comment by jbottum** [comment]
line 149 , perhaps....each of the working group leads

**Comment by annajung** [comment]
@jbottum created a PR to follow up on his suggestions https://github.com/kubeflow/blog/pull/136

**Comment by annajung** [comment]
> line 91, If you are interested in improving Kubeflow documentation and forming a Kubeflow Documentation Team, join us at the next community meeting to introduce yourself to the community. Comment - We have not discussed forming a Kubeflow Documentation Team. I think its a good idea, but probably should float it in a community meeting. Additionally, I might have been a little more aggressive and add or reword i.e. We are asking that all companies, who are using Kubeflow, to please proactively contribute at least one Pull Request on documentation in Kubeflow 1.8. We request that you please add your PRs to the appropriate Working Group Meeting notes, and review it in the next Working Group meeting.

@jbottum This was discussed during the last community meeting and my sense of the meeting was that people were in favor of it if there was folks who were willing to lead. If that's not the case, feel free to remove this section in your follow up PR

**Pull request merged**